pkgpanda/__init__.py

```diff
@@ -545,7 +545,7 @@ def add_user(self, username, group):
             '-c', 'DCOS System User',
         ]
 
-        if group is not None:
+        if group is not None and group != username:
```

Comment:
Also, does it make sense to do a bit of refactoring here to use the variable name `groupname`, for consistency with `username`?